refinery/core/models.py

```diff
@@ -1483,6 +1504,14 @@ def get_input_file_uuid_list(self):
             input_file_uuid_list.append(cur_fs_uuid)
         return input_file_uuid_list
 
+    def data_sets_query(self):
+        analysis_facet_name = '{}_{}_{}_s'.format(
+            NodeIndex.ANALYSIS_UUID_PREFIX,
+            self.data_set._get_study().id,
```

Comment:
No, this is correct: We're in [`Analysis`](https://github.com/refinery-platform/refinery-platform/pull/2009/files/f710cd4a4b2a9695155940522498c730c85d34d2#diff-10b1a3c979459ad2810c191d2321daa3L1121) and need to go up to get the `DataSet`